### ISSUE TITLE
Spike: test pyGeoAPI

### ISSUE THREAD

**Issue by billgeo** [linz/geostore#2233]
So that we can decide which [STAC API / Indexing tools](https://toitutewhenua.atlassian.net/wiki/spaces/LGDL/pages/127796416/STAC+API+Options) we want to use we want to take PyGeoAPI for a test run

Timeboxed to 3 days.

- [x]     create a new page [from the Confluence page template](https://toitutewhenua.atlassian.net/wiki/spaces/LGDL/pages/127865425/Solution+-+Test+Run+Template)
- [x]     Read Github history, issues, pulse, contributors etc
- [x]     Read the documentation
- [ ]     Find existing implementations
- [ ]     Possibly, test run the software
- [ ]     Possibly, ingest some of our metadata
- [x]     assess it against the criteria and update the confluence page with your findings

**Comment by Jimlinz** [comment]
tldr; Not really a good fit for geostore. Have not dig further as there are other better options available. 

Short write up below:
https://toitutewhenua.atlassian.net/wiki/spaces/LGDL/pages/133791762/pygeoapi+review

**Comment by billgeo** [comment]
Thanks @Jimlinz . Looks good. Closing this one. I updated the decision in the API Options page too to reflect this.
